### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by oilbeater**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/contrib/1730)

### CODE REVIEW

**Comment by oilbeater** [comment]
I signed it!

File: ingress/controllers/nginx/README.md [hunk 1]
```diff
@@ -141,7 +141,7 @@ Check the [example](examples/tls/README.md)
 
 ### Default SSL Certificate
 
-NGINX provides the option serve rname [_](http://nginx.org/en/docs/http/server_names.html) as a catch-all in case of requests that do not match one of the configured server names. This configuration works without issues for HTTP traffic. In case of HTTPS NGINX requires a certificate. For this reason the Ingress controller provides the flag `--default-ssl-certificate`. The secret behind this flag contains the default certificate to be used in the mentioned case.
+NGINX provides the option server name [server name](http://nginx.org/en/docs/http/server_names.html) as a catch-all in case of requests that do not match one of the configured server names. This configuration works without issues for HTTP traffic. In case of HTTPS NGINX requires a certificate. For this reason the Ingress controller provides the flag `--default-ssl-certificate`. The secret behind this flag contains the default certificate to be used in the mentioned case.
```
Comment (ingvagabund) [inline ingress/controllers/nginx/README.md]: s/server name [server name]/[server name]

**Comment by ingvagabund** [comment]
Up to the one nit it looks good. Thanks for the patch.

**Comment by bprashanth** [comment]
Thanks, lgtm

**Pull request merged**